compliance/unapproved_instance_types/unapproved_instance_types.pt

```diff
@@ -0,0 +1,191 @@
+name "Unapproved instance types"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report on any instances that are running using instance types that are not approved. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/compliance/approved_instance_types) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
path to README is using the wrong directory

Reply:
I have made the required changes.